manifests/polkit.pp

```diff
@@ -0,0 +1,30 @@
+# Add a rule file allowing members of a group to use libvirt
+#
+# @param ensure Create or destroy the rules file
+#
+# @param content An arbitrary string of javascript polkit configuration
+#
+# @param priority Priority of the file to be created
+#
+# @param rulesd Location of the poklit rules directory
+#
+class libvirt::polkit (
+  Enum['present','absent'] $ensure   = 'present',
+  String                   $group    = 'virtusers',
```

Comment:
Do we want to allow groups and users?

Also, this should be an `Array[String]` so that you can specify multiple groups at the same time.